src/main/java/frc/robot/subsystems/DriveTrain.java

```diff
@@ -87,6 +87,9 @@ public DriveTrain() {
 
     m_odometry = new DifferentialDriveOdometry(Rotation2d.fromDegrees(-getGyroAngle()), new Pose2d(
         Constants.ROBOT_START_X, Constants.ROBOT_START_Y, Rotation2d.fromDegrees(Constants.ROBOT_START_ANGLE)));
+
+    resetDriveTrain();
+    //TODO : Call reset drivetrain at the beginning of anyg auto command once they get merged in
```

Comment:
Why not just call this in `autonomousInit()`? That way we're sure it gets called at the start.

Reply:
If I'm not mistaken, calling this in `autonomousInit` would require the `m_driveTrain` to be publicly accessible. I'm not sure if this is in line with the new WPILIB design philosophy. As long as you're ok with that, I can put it in.

Reply:
That's true, perhaps a corresponding function (eg `setupAuto()`) in RobotContainer would be better. We're going to need to pass subsystems to the autonomous command group anyways, so we'll need some mechanism to access the subsystems at enable time. Adding in this reset call seems like it could go alongside the command group setup.

Reply:
Resolved in b26f458